app/models/cloud_volume.rb

```diff
@@ -23,6 +23,9 @@ class CloudVolume < ApplicationRecord
   has_many   :vms, :through => :hardwares, :foreign_key => :vm_or_template_id
   has_many   :volume_mappings, :dependent => :destroy
 
+  supports_not :backup_create, :description  => 'CloudVolume backup creation'
```

Comment:
I think we're going to have to i18n every one of these, and since it includes the model name, there's a special way to do that.

Reply:
Something like `N_('%(model) backup creation')`, but then I don't know how to apply that later when it's actually evaluated.

@mzazrivec Can you help here?

Reply:
It is good to get an I18N understanding for these types of situations.

Reply:
Deleted descriptions.  So, this is not needed for this PR.

Reply:
I'd say
```ruby
supports_not :backup_create, :description  => N_('Cloud Volume backup creation')
```

is correct in this case.  You don't need to do `N_("%(model) backup creation")` here, since you already know the model and you want to avoid string arithmetic in i18n strings as much as possible.